ONSdigital/census-rm-acceptance-tests-2021
# Verify new case service emits case created events to RH

# Motivation and Context

We created a new case service which handles the emitting of the case created events to RH and we need to verify that this is working end-to-end.

# What has changed



* We put the samples onto a new queue as JSON messages
* We check that the RH queue has the expected number of case created messages on receiving end

# How to test?



Run the tests against the updated docker-dev (see link below)

https://github.com/ONSdigital/census-rm-docker-dev/pull/10

# Links



https://trello.com/c/jRjqBbgh/576-emit-casecreated-event-8
